build/common.sh

```diff
@@ -573,23 +542,47 @@ function kube::build::ensure_data_container() {
     # We want this to run as root to be able to chown, so non-root users can
     # later use the result as a data container.  This run both creates the data
     # container and chowns the GOPATH.
+    #
+    # The data container creates volumes for all of the directories that store
```

Comment:
Can't we just use the local _output/local/go/pkg dir to cache incremental builds, and get rid of the data container entirely?  We're using docker as a way to get a hermetic, reproducable build with known-good tools.  The source itself is in my local dir.  The data container is a leftover from a bygone era.